bridges/NovayaGazetaEuropeBridge.php

```diff
@@ -0,0 +1,119 @@
+<?php
+class NovayaGazetaEuropeBridge extends BridgeAbstract
+{
+
+	const MAINTAINER = 'sqrtminusone';
+	const NAME = 'Novaya Gazeta Europe Bridge';
+	const URI = 'https://novayagazeta.eu';
+
+	const CACHE_TIMEOUT = 3600; // 1 hour
+	const DESCRIPTION = 'Returns articles from Novaya Gazeta Europe';
+
+	const PARAMETERS = array(
+		'' => array(
+			'language' => array(
+				'name' => 'Language',
+				'type' => 'list',
+				'defaultValue' => 'ru',
+				'values' => array(
+					'Russian' => 'ru',
+					'English' => 'en',
+				)
+			)
+		)
+	);
+
+	public function collectData()
+	{
+		$url = 'https://novayagazeta.eu/api/v1/get/main';
+		if ($this->getInput('language') != 'ru') {
+			$url .= '?lang=' . $this->getInput('language');
+		}
+
+		$json = getContents($url);
+		$data = json_decode($json);
+
+		foreach ($data->records as $record) {
+			foreach ($record->blocks as $block) {
+				if (!property_exists($block, 'date')) {
+					continue;
+				}
+				$body = '';
+				if (property_exists($block, 'body') && $block->body !== null) {
+					$body = self::convertBody($block->body, $block->lead);
+				} else {
+					$record_json = getContents("https://novayagazeta.eu/api/v1/get/record?slug={$block->slug}");
+					$record_data = json_decode($record_json);
+					$body = self::convertBody($record_data->record->body, $record_data->record->lead);
+				}
+				$title = strip_tags($block->title);
+				if (property_exists($block, 'subtitle') && mb_strlen($block->subtitle) > 0) {
+					$title .= '. ' . strip_tags($block->subtitle);
+				}
+				$item = array(
+					'uri' => self::URI . '/articles/' . $block->slug,
+					'title' => $title,
+					'author' => join(', ', array_map(function ($author) {
+						return $author->name;
+					}, $block->authors)),
+					'timestamp' => $block->date / 1000,
+					'categories' => $block->tags,
+					'content' => $body
+				);
+				$this->items[] = $item;
+			}
+		}
+		usort($this->items, function ($item1, $item2) {
+			return $item1['timestamp'] < $item2['timestamp'];
```

Comment:
```suggestion
			return  $item2['timestamp'] <=> $item1['timestamp'];
```

This is because:
` Deprecated: usort(): Returning bool from comparison function is deprecated, return an integer less than, equal to, or greater than zero in bridges/NovayaGazetaEuropeBridge.php on line 68`

Reply:
Done.

But how to get these warnings? I can't find them in actions or PHP output

Reply:
Do you have DEBUG mode enabled? You can enable it by creating a file called DEBUG in the rssbridge folder. Otherwise, these warnings could be a difference between php versions.

Reply:
Yes, I've been testing on php v7.4.26, because that's the one packaged with Guix. And it looks like this particular warning comes from php >= 8.